app/src/main/java/com/yterletskyi/happyfriend/features/settings/ui/SeparateLineItem.kt

```diff
@@ -0,0 +1,31 @@
+package com.yterletskyi.happyfriend.features.settings.ui
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import androidx.core.view.get
+import androidx.recyclerview.widget.RecyclerView
+
+class SeparateLineItem : RecyclerView.ItemDecoration() {
+
+    private val paint = Paint().apply {
+        color = Color.parseColor("#d3d3d3")
+    }
+
+    override fun onDraw(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
+        val width = parent[parent.childCount - 1].width.toFloat()
+        val y = parent[parent.childCount - 1].y
```

Comment:
`parent[parent.childCount - 1]` looks to be a duplicate part in 16 and 17 lines. Can we extract it to some variable for example `val lastChild = parent[parent.childCount - 1]` and then use it like `val width =lastChild.width.toFloat()` and `val y = lastChild.y` ?